src/components/vehicleDataGroup/index.stories.mdx

```diff
@@ -0,0 +1,56 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import VehicleDataGroupField from './VehicleDataGroupField.tsx';
+import { CloseIcon } from '../icons';
+
+import VehicleDataGroup from './';
+
+<Meta
+  title="Components/Data Display/VehicleDataGroup"
+  component={VehicleDataGroup}
+/>
+
+export const ItemLabel = () => (
+    <>
+    <CloseIcon />
+    Some value
+    </>
+)
+
+export const ItemValue = () => (
+  <>
+    <CloseIcon />
+    Some value
+  </>
+)
+
+export const Template = (args) => {
+
+const label = 'VehicleDataGroupField label';
+const value = 'VehiculeDataGroupField Value';
+
+return (
+
+<VehicleDataGroup {...args} title={args.title} w={args.w}>
+  <VehicleDataGroupField label={label} value={value} />
+  <VehicleDataGroupField label={label} value={value} />
+  <VehicleDataGroupField label={label} value={value} />
+  <VehicleDataGroupField label={<ItemLabel />} value={<ItemValue />} />
+</VehicleDataGroup>
+) };
+
+# GroupLabel
+
+## Overview
+
+<Canvas>
+  <Story
+    name="GroupLabel"
+    args={{
+      title: 'Vehicle Group Data',
+      w: '400px',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
```

Comment:
I will recommend to add the next cases as stories:

1. Overview
2.  Label and value with icon.
3. Label and value without icon.
4. Group without title

We try to be more specific with the. stories and show the different cases. 
You can also add the ArgsTable in the stories to be able to interact. with its properties.

Thank you!

Reply:
> One thing regarding the mobile design. Maybe you need to check with the designer about the case of. mobile
> 
> **Your branch**
> 
> <img alt="Screenshot 2022-11-29 at 13 38 14" width="401" src="https://user-images.githubusercontent.com/37380787/204531410-160486e3-9b06-4766-9a5b-2fd841eb7129.png">

It is because width is hardcoded in story to 440px.
If you remove 'w' prop it will be responsive...

Reply:
Maybe you could do it responsive as we do for the article teaser. Then you can add it as another story. What do you think?